theano/gof/cmodule.py

```diff
@@ -2009,11 +2009,12 @@ def join_options(init_part):
         # in the key of the compiled module, avoiding potential conflicts.
 
         # Figure out whether the current Python executable is 32
-        # or 64 bit and compile accordingly. This step is ignored for ARM
-        # architectures in order to make Theano compatible with the Raspberry
-        # Pi, and Raspberry Pi 2.
+        # or 64 bit and compile accordingly. This step is ignored for 
+        # ARM (32-bit and 64-bit) architectures in order to make
+        # Theano compatible with the Raspberry Pi, Raspberry Pi 2, or 
+        # other systems with ARM processors.
         if (not any(['arm' in flag for flag in cxxflags]) and
-                'arm' not in platform.machine()):
+            not any(arch in platform.machine() for arch in ['arm', 'aarch'])):
```

Comment:
For some reason, this line need 4 extra space at the start, as the old line.

Those 2 small changes should make the tests pass. 

thanks